packages/media-library/src/api/fetch.ts

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
I wonder how you feel about the prospect of decoupling the data layer from `lib/wpcom`, then abstracting it as a separate package and using that package for performing the requests? After all, I'd expect that the media library itself shouldn't have to reinvent the wheel as it comes to data needs - instead, it should step on something already built. What do you think?

Reply:
What do you define as the data layer? It feels like that's what the `Client` is?

Reply:
I was talking about the [data layer](https://github.com/Automattic/wp-calypso/tree/master/client/state/data-layer) that much of Calypso state already uses.